packages/mds-utils/tests/csv-stream.spec.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2022 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ApiResponse } from '@mds-core/mds-api-server'
+import { Nullable } from '@mds-core/mds-types'
+import { Cursor } from 'typeorm-cursor-pagination'
+import { csvStreamFromRepository } from '../csv-stream'
+
+// TODO either convert this test file to mocha, convert the others to jest, or do something weird
```

Comment:
Opinions sought

Reply:
Probably going to move the whole thing to `mds-api-helpers` instead of `mds-utils`

Reply:
Moved, now to pull in Russen's changes from russen/features/PLAT-372-convert-mds-api-to-jest